### PR TITLE
Update root README.md

### PR DESCRIPTION
**Pull request by jvaclavik**
See how it looks like here: https://github.com/kiwicom/margarita/tree/docs/update-root-readme

 
screencapture-github-kiwicom-margarita-blob-docs-update-root-readme-README-md-2019-05-28-17_39_54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,72 @@
...
+## How it works
+
+<img src="./docs/assets/schema.png" alt="Schema" title="Schema" />
+
+## Links
+
+- 🎮 [GraphQL Playground](https://p2kwd3i3a8.execute-api.eu-central-1.amazonaws.com/staging/graphql)
+- 🚀 [GraphQL Voyager](https://margarita-graphql-voyager.now.sh/)
+- 🕸️ [Margarita web](http://margarita.kiwi.com)
+- 🥝 [Open source projects at Kiwi.com](https://margarita-graphql-voyager.now.sh/)
```
Comment (JoseDuSV) [inline README.md]: URL points to interactive schema instead of Kiwi.com repo.

Comment (jvaclavik) [reply README.md]: Thank you! Fixed

File: README.md [hunk 2]
```diff
@@ -1,33 +1,68 @@
...
-- [Getting Started](https://kiwicom.github.io/margarita/docs/getting-started)
-- [Technical Overview](https://kiwicom.github.io/margarita/docs/technical-overview)
+- React Native
+- Expo
+- React Native Web
+- Next.js
+- Relay
+- Flow
+
+More detailed info about our devstack can be found in [Technical Overview docs](https://margarita.kiwi.com).
```
Comment (JoseDuSV) [inline README.md]: One more link change. Should be https://kiwicom.github.io/margarita/docs/technical-overview

File: README.md [hunk 3]
```diff
@@ -1,33 +1,68 @@
...
 
-The full documentation for **Margarita** can be found on our [website](https://kiwicom.github.io/margarita/docs/).
+## Devstack
 
-## 💻 Developing
+This project follows react-native-web philosophy of **Write Once, Render Anywhere**.
 
-- [Getting Started](https://kiwicom.github.io/margarita/docs/getting-started)
-- [Technical Overview](https://kiwicom.github.io/margarita/docs/technical-overview)
+- React Native
```
Comment (JoseDuSV) [inline README.md]: What about making those clickable?
https://facebook.github.io/react-native/
https://expo.io/
https://github.com/necolas/react-native-web
https://nextjs.org/
http://facebook.github.io/relay/
https://flow.org/

File: README.md [hunk 4]
```diff
@@ -1,33 +1,68 @@
...
+<h4 align="center">Multi-platform <span title="travel">✈️</span> travel app  for iOS, Android and Web<br></h4>
 
-# Margarita
+ <p align="center">built with <strong>React Native, GraphQL and <a href="https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/" target="_blank">Tequila API</a></strong></p>
 
-> Showcasing the power of [Tequila](https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/)
+## Demo
 
-This project is a living example of a client app you could start building with Tequila.
+Live demo: https://kiwicom-margarita.netlify.com (web version)
```
Comment (JoseDuSV) [inline README.md]: We can also consider adding Expo link for mobile version https://expo.io/@kiwicom-margarita-public/tequila-client

Comment (jvaclavik) [reply README.md]: Good idea. 
We should change this: 
Screenshot 2019-05-29 at 10 08 23

File: README.md (hunk 2 above)
Comment (jvaclavik) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (jvaclavik) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (jvaclavik) [reply README.md]: Done

Comment (JoseDuSV) [reply README.md]: Definitely. I've already created separate issue for it https://github.com/kiwicom/margarita/issues/849

**Pull request merged**